ethereum-network-partition-test/main_test.go

```diff
@@ -85,13 +85,17 @@ const (
 	retriesAttempts      = 20
 	retriesSleepDuration = 10 * time.Millisecond
 
-	newlineChar = "\n"
+	updateScriptStarlarkTemplate  = `plan.update_service(service_id = "%s", config = UpdateServiceConfig(subnetwork = "%s"))`
+	setConnectionStarlarkTemplate = `plan.set_connection(subnetworks = ("%s", "%s"), config = ConnectionConfig(packet_loss_percentage = %f))`
```

Comment:
you could use `kurtosis.connection.BLOCKED` or `ALLOWED` here, since you're using either 0 or 100%

Reply:
Nice! I wonder if we should update the docs on it to contain that example

Reply:
Yeah we should at least mention those constants somewhere in the docs. I suspect this `kurtosis.XX` thing will grow overtime, I wonder if we should have a dedicated section for it